npm/private/npm_import.bzl

```diff
@@ -682,22 +649,21 @@ def _npm_import_rule_impl(rctx):
             for bin_name in bins:
                 sanitized_bin_name = _sanitize_bin_name(bin_name)
                 bin_struct_fields.append(
-                    """        {bin_name} = lambda name, **kwargs: _{bin_name}_internal(name, link_root_name = link_root_name, **kwargs),
-        {bin_name}_test = lambda name, **kwargs: _{bin_name}_test_internal(name, link_root_name = link_root_name, **kwargs),
-        {bin_name}_binary = lambda name, **kwargs: _{bin_name}_binary_internal(name, link_root_name = link_root_name, **kwargs),
+                    """        {bin_name} = {bin_name},
```

Comment:
maybe leave a TODO to cleanup eihter this or the macros above? Seems weird to have both APIs